js/exmo.js

```diff
@@ -472,25 +474,29 @@ module.exports = class exmo extends Exchange {
         //         ],
         //     }
         //
-        const result = {
-            'info': cryptoList,
-            'withdraw': {},
-            'deposit': {},
-        };
+        const result = {};
```

Comment:
@AlgoQ Do we need this extra call?
![image](https://user-images.githubusercontent.com/43336371/198240228-24014ff5-7561-4d55-be97-981a5ae983c7.png)

Can you try to replace this with `this.currencies` ?